Minesweeper.java

```diff
@@ -81,24 +105,57 @@ private void run() {
             setVisible(true);
         }
 
+        public void revealEmptyCells(int row, int col) {
+            // Check bounds to avoid array out-of-bounds exceptions
+            if (row < 0 || col < 0 || row >= Game.cells.length || col >= Game.cells[0].length) {
+                return;
+            }
+            
+            Cell cell = Game.cells[row][col];
+            
+            // Check if the cell is already revealed or contains a mine
+            if (cell.isRevealed || cell.isBomb) {
+                return;
+            }
+            
+            // Reveal the cell
+            Game.cells[row][col].reveal();
+            
+
+            // If this cell has no adjacent mines, recursively reveal its neighbors
+            if (cell.getNeighboringBombs(Game.cells[row][col]) == 0) {
```

Comment:
just use the `Game` method for this
```suggestion
            if (computeNeighboringBombs(this.cells[row][col]) == 0) {
```